Documentation/coding-guidelines/interop-pinvokes.md

```diff
@@ -95,42 +95,49 @@ Guids are usable directly in signatures. When passed by ref they can either be p
+| 8     | `BYTE`           | `unsigned char`    | `byte`   |                                      |
+| 8     | `CHAR`           | `char`             | `sbyte`  |                                      |
+| 8     | `UCHAR`          | `unsigned char`    | `byte`   |                                      |
+| 16    | `SHORT`          | `short`            | `short`  |                                      |
+| 16    | `CSHORT`         | `short`            | `short`  |                                      |
+| 16    | `USHORT`         | `unsigned short`   | `ushort` |                                      |
+| 16    | `WORD`           | `unsigned short`   | `ushort` |                                      |
+| 16    | `ATOM`           | `unsigned short`   | `ushort` |                                      |
+| 32    | `INT`            | `int`              | `int`    |                                      |
+| 32    | `LONG`           | `long`             | `int`    |                                      |
+| 32    | `ULONG`          | `unsigned long`    | `uint`   |                                      |
+| 32    | `DWORD`          | `unsigned long`    | `uint`   |                                      |
+| 64    | `QWORD`          | `__int64`          | `long`   |                                      |
+| 64    | `LARGE_INTEGER`  | `__int64`          | `long`   |                                      |
+| 64    | `LONGLONG`       | `__int64`          | `long`   |                                      |
+| 64    | `ULONGLONG`      | `unsigned __int64` | `ulong`  |                                      |
+| 64    | `ULARGE_INTEGER` | `unsigned __int64` | `ulong`  |                                      |
+| 32    | `HRESULT`        | `long`             | `int`    |                                      |
+| 32    | `NTSTATUS`       | `long`             | `int`    |                                      |
+
+
+The following types, being pointers, do follow the width of the platform. Use `IntPtr`/`UIntPtr` for these.
+
+| Signed Pointer Types (use `IntPtr`) | Unsigned Pointer Types (use `UIntPtr`) |
+|:------------------------------------|:---------------------------------------|
+| `HANDLE`                            | `WPARAM`                               |
+| `HWND`                              | `UINT_PTR`                             |
+| `HINSTANCE`                         | `ULONG_PTR`                            |
+| `LPARAM`                            | `SIZE_T`                               |
+| `LRESULT`                           |                                        |
+| `LONG_PTR`                          |                                        |
+| `INT_PTR`                           |                                        |
+
+A Windows `PVOID` which is a C `void*` can be marshaled as either `IntPtr` or `UIntPtr`. Generally, it doesn't matter.
+
+[Windows Data Types](http://msdn.microsoft.com/en-us/library/aa383751.aspx "MSDN")
```

Comment:
Nit: You can update the links to point to docs while you are on it to skip the forward.